Instagram/Instagram/Source/Screen/Auth/Controller/UserPasswordViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  UserPasswordViewController.swift
+//  Instagram
+//
+//  Created by 소연 on 2022/04/04.
+//
+
+import UIKit
+
+import RxSwift
+
+import SnapKit
+import Then
+
+final class UserPasswordViewController: UIViewController {
+    
+    // MARK: - Properties
+
+    var disposeBag = DisposeBag()
+    
+    private lazy var navigationBar = IDSNavigationBar(self,
+                                                      view: .auth,
+                                                      backButtonIsHidden: false,
+                                                      closeButtonIsHidden: true)
+    
+    private lazy var rootView = SignUpView()
+    
+    var userName: String = "" 
+    
+    // MARK: - Life Cycle
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        navigationController?.isNavigationBarHidden = true
+    }
```

Comment:
지금은 한줄밖에 없지만, 나중에 네비게이션 관련 코드가 더 들어올 수 있으니 함수로 빼주는것도 좋을 것 같아!
```suggestion
    override func viewWillAppear(_ animated: Bool) {
        super.viewWillAppear(animated)
        setNavigationBar()
    }
    
    func setNavigationBar() {
        navigationController?.isNavigationBarHidden = true
    }
```